### DISCUSSION TITLE
ghostscript suddenly dropped x11 support

### DISCUSSION THREAD

**Discussion by denismm** [Homebrew/discussions#1479] [Everyday usage]
After a bad experience losing gv support in homebrew, I've been avoiding doing any homebrew updates so as not to break anything.  This evening I installed graphviz and suddenly ghostscript no longer supports x11 as a device.  As far as I can tell from a brief search it looks like x11 support was dropped from ghostview's formula 2 years ago as part of the "drop all options" activity.

I found some discussions about x11 support being added back in last year - is that something that can happen for ghostscript (and possibly returning gv to the supported list)?

**Comment by SMillerDev** [comment]
> I found some discussions about x11 support being added back in last year - is that something that can happen for ghostscript (and possibly returning gv to the supported list)?

That depends on what ghostscript needs for this. But you can make a pull request here: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/ghostscript.rb and I'll happily review it. (https://docs.brew.sh/Updating-Software-in-Homebrew)

**Reply by denismm** [reply]
I was able to get https://github.com/johnhcc/homebrew-gs-x11 working - my PR would basically to bring both gs and gv from this cask into the mainline.
